doc/installation.md

```diff
@@ -32,6 +34,15 @@ There are six major components to a Kazoo system, they can all be installed on o
 * Monster-UI
   * Provides the Kazoo web interface for configuration and monitoring of the system.
 
+## Cluster design considerations
+
+There are a few concerns that should be planned for when designing a Kazoo cluster.
+
+* Design in enough bigcouch servers from the start
+  * It is very difficult to add additional database servers once the cluster has been deployed, as such you should plan in advance and add the maximum number of bigcouch servers that you forsee needing over the lifetime of the cluster from the start.
```

Comment:
This seems too extreme for most folks. Maybe tone it down a bit to say something like "Plan to provision enough database servers to cope with your expected growth over the next year or two." Something along these lines, anyway.